tests/input/draft-v3-features.xml

```diff
@@ -32,6 +32,8 @@
         </postal>
       </address>
     </author>
+    <author asciiFullname="Armin Bergstrom" fullname="Armin Bergström">
```

Comment:
This example should never happen.  "asciiFullname" is not about ASCII at all.  Armin's name is fully Latin so doesn't need an asciiFullname.

Reply:
Thanks, I'll correct this.